services/socketIOSetup.js

```diff
@@ -9,6 +9,7 @@ import {
   getStudent,
   addStudentToClassroom,
   remStudentFromClassroom,
+  unPairStudentChat,
```

Comment:
I'd prefer we not capitalize the P as so far we've referred to it as unpair.  The socket event says `"unpair student chat"` with no capitalized P.
```suggestion
  unpairStudentChat,
```